crates/http-api-bindings/src/rate_limit.rs

```diff
@@ -11,28 +11,23 @@ use futures::stream::BoxStream;
 use ratelimit::Ratelimiter;
 use tabby_inference::{ChatCompletionStream, CompletionOptions, CompletionStream, Embedding};
 
-fn new_rate_limiter(rpm: u64) -> anyhow::Result<Ratelimiter> {
+fn new_rate_limiter(rpm: u64) -> Ratelimiter {
     Ratelimiter::builder(rpm, Duration::from_secs(60))
         .max_tokens(rpm)
         .initial_available(rpm)
         .build()
-        .map_err(|e| {
-            anyhow::anyhow!(
-                "Failed to create ratelimiter, please check the rate limit configuration: {}",
-                e,
-            )
-        })
+        .expect("Failed to create RateLimiter, please check the HttpModelConfig.rate_limit configuration")
```

Comment:
```suggestion
        .expect("Failed to create RateLimiter, please check the model.*.http.rate_limit.rate_limit configuration")
```

Perhaps we could assist users by specifying the configuration details directly like this?

Reply:
ack.